app/views/user.py

```diff
@@ -263,13 +334,25 @@ class ResetPassword(Resource):
 
     def post(self):
         """Reset user password.
-
-        Returns:
-            A success message to indicate successful logout.
-            An error message if username does not exist.
-            An error message if no data is provided.
-            An error message if the password is less than 6 characters.
-
+        ---
+        tags:
+            - User authentication and authorization
+        parameters:
+            -   in: body
+                name: body
+                schema:
+                    $ref: '#/definitions/User'
+        responses:
+            200:
+                description: Password reset successfully
+                content:
+                    application/json:
+                        schema:
+                            type: object
+                            properties:
+                                response_message:
+                                    type: string
+                                    description: response message to show password has been reset successful
         """
         req_data = request.get_json()
         email = req_data['email']
```

Comment:
Then it would be better to use `email = req_data.get('email')`  to avoid errors if one of the keys is missing in the request